docs/csharp/tour-of-csharp/expressions.md

```diff
@@ -39,7 +39,7 @@ The following summarizes C#’s operators, listing the operator categories in or
 	- `new T(...){...}`: Object creation with initializer
 	- `new {...}`:  Anonymous object initializer
 	- `new T[...]`: Array creation
-	- `typeof(T)`: Obtain System.Type object for `T`
+	- `typeof(T)`: Obtain `System.Type` object for `T`
```

Comment:
nit: You could also style System.Type as [System.Type](xref:System.Type) and that would create a link to the API doc.